src/pages/projectSettings/tabs/ViewsAndFiltersTab/ViewsAndFiltersTab.tsx

```diff
@@ -0,0 +1,35 @@
+import { useMemo } from "react";
+import { SpruceForm } from "components/SpruceForm";
+import { ProjectSettingsTabRoutes } from "constants/routes";
+import {
+  usePopulateForm,
+  useProjectSettingsContext,
+} from "pages/projectSettings/Context";
+import { getFormSchema } from "./getFormSchema";
+import { TabProps } from "./types";
+
+const tab = ProjectSettingsTabRoutes.ViewsAndFilters;
+
+export const ViewsAndFiltersTab: React.VFC<TabProps> = ({ projectData }) => {
+  const { getTab, updateForm } = useProjectSettingsContext();
+  const { formData } = getTab(tab);
+
+  const initialFormState = projectData;
+  usePopulateForm(initialFormState, tab);
+
+  const onChange = updateForm(tab);
+
+  const { fields, schema, uiSchema } = useMemo(() => getFormSchema(), []);
+
+  if (!formData) return null;
+
+  return (
+    <SpruceForm
+      fields={fields}
+      formData={formData}
+      onChange={onChange}
+      schema={schema}
+      uiSchema={uiSchema}
+    />
+  );
+};
```

Comment:
At some point maybe we should make a generic project settings component that handles all this stuff 😅 but looks good for now!

Reply:
I shall file a ticket :)